src/lib/core/ripple/ripple-ref.ts

```diff
@@ -5,6 +5,9 @@ import {RippleConfig, RippleRenderer} from './ripple-renderer';
  */
 export class RippleRef {
 
+  /** Whether the ripple finished it's fade-in animation. */
+  fadedIn: boolean = false;
```

Comment:
Wouldn't it be more flexible to have this as a string (custom type) along the lines of `state`. That way we can expand it later to check for other cases, if we need to.